### ISSUE TITLE
Kafka: Consumer configuration "auto.offset.reset"

### ISSUE THREAD

**Issue by Amakaev** [dotnetcore/CAP#822]
Hey, why is `"auto.offset.reset"` hardly set's to `earliest`? Can it be configured to `latest`?
I don't wanna get all topics messages on attaching new consumer group.
https://github.com/dotnetcore/CAP/blob/641d4acb266ac0538bbba4c351d9a9d041e72862/src/DotNetCore.CAP.Kafka/KafkaConsumerClient.cs#L109-L117

**Comment by yang-xiaodong** [comment]
Hello,

Thanks for your feedback , We now support rewriting the default configuration of Kafka consumer, Example:

```C#
 x.UseKafka(z =>
  {
      z.Servers = "localhost:9092";
      z.MainConfig.Add("auto.offset.reset", "latest");
  });
```

**Comment by yang-xiaodong** [comment]
Issue was fixed in 5.0.1 and released to NuGet a few minutes ago
